src/main/scala/interfaces/Verification.scala

```diff
@@ -113,3 +114,18 @@ case class SiliconVariableCounterexample(internalStore: Store, nativeModel: Mode
   }
 }
 
+case class SiliconMappedCounterexample(internalStore: Store, heap: Iterable[Chunk], oldHeaps: State.OldHeaps, nativemodel: Model) extends SiliconCounterexample {
+  val converter = Converter(nativemodel, internalStore, heap, oldHeaps)
+  
+  override def toString = {
+    var buf : String = ""
+    buf += converter.modelAtLabel.map(x => "model at label: " + x._1 + "\n" + x._2.toString).mkString("\n")
+    buf += "\nfinally: \n" + converter.extractedModel.toString
+    buf
+  }
+  val model : Model = nativemodel
```

Comment:
Style: separate defs and val with newlines

Reply:
I think this comment might still apply in the current version @cedihegi